### ISSUE TITLE
Clarify the relationship between Form, Form::op and protocol bindings

### ISSUE THREAD

**Issue by lu-zero** [w3c/wot-thing-description#1782]
The protocol bindings usually extend Form in two ways:
- Adding a default verb-to-operation mapping
- Providing explicit verbs to use to override the mapping

If the default mapping is used and only `op` is different in Forms, they can be collapsed in a single one with an Array.

Would be good to either have an explicit `verb` field that is a single-entry to structurally prevent having invalid Forms with an explicit verb AND `op` in Array from or add a third form for `op` as a `Map<opstring, verb>`.

Originally discussed in https://github.com/w3c/wot-binding-templates/issues/194

**Comment by benfrancis** [comment]
Just a thought, but not all protocols use "verbs", in fact possibly only HTTP and CoAP follow that pattern (e.g. See the [MQTT](https://w3c.github.io/wot-binding-templates/bindings/protocols/mqtt) and [Modbus](https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/index.html) binding documents).

Non-default bindings are also rarely as simple as just one vocabulary term, so you'd actually need a map of maps. At that point it may just be simpler/clearer to provide separate forms.

**Comment by relu91** [comment]
Yes, I was quite surprised how the concept of verbs could be adapted to MQTT and Modbus too. Still, I'm a little bit unsure if this can be transposed to also any future protocol that we could support. 

I probably need to think more about this issue, because I don't dislike even a more drastic solution of deprecating the op-Array form.
